presentation/src/main/java/com/keeply/presentation/core/components/Buttons.kt

```diff
@@ -0,0 +1,236 @@
+package com.keeply.presentation.core.components
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.defaultMinSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.ProvideTextStyle
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.semantics.role
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.keeply.presentation.core.theme.KeeplyTheme
+import com.keeply.presentation.core.theme.LocalTypography
+import com.keeply.presentation.core.theme.neutral300
+import com.keeply.presentation.core.theme.neutral400
+import com.keeply.presentation.core.theme.neutral500
+import com.keeply.presentation.core.theme.neutral800
+import com.keeply.presentation.core.theme.neutralBlack
+import com.keeply.presentation.core.theme.neutralWhite
+
+@Composable
+fun KeeplyButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    icon: Painter? = null,
+    text: String,
+    textStyle: TextStyle = KeeplyTheme.typography.button02Suit,
+    buttonSize: KeeplyButtonSize = KeeplyButtonSize.MEDIUM,
+    buttonStyle: KeeplyButtonStyle = KeeplyButtonStyle.PRIMARY
+) {
+    Surface(
+        onClick = onClick,
+        modifier = modifier
+            .semantics { role = Role.Button },
+        enabled = enabled,
+        shape = RoundedCornerShape(size = 4.dp),
+        color = if (enabled) buttonStyle.containerColor else buttonStyle.disabledContainerColor,
+        contentColor = if (enabled) buttonStyle.contentColor else buttonStyle.disabledContentColor // 필요없으면 제거
+    ) {
+        ProvideTextStyle(value = textStyle) {
+            Row(
+                Modifier
+                    .defaultMinSize(
+                        minWidth = buttonSize.width,
+                        minHeight = buttonSize.defaultHeight
+                    )
+                    .padding(buttonSize.contentPadding),
+                horizontalArrangement = Arrangement.Center,
```

Comment:
p3: 아이템 사이의 간격도 Icon 하단의 Spacer로 추가하지 않고 spacedBy를 활용하면 조금 더 깔끔한 코드가 될 것 같아요.
```suggestion
                horizontalArrangement = Arrangement.spacedBy(4.dp, Alignment.CenterHorizontally),
```

Reply:
좋은 방법이네요 수정했습니다! https://github.com/DDD-Community/DDD-12-Keeply-Android/pull/15/commits/c28350dfb1f2039e473ea25fd06783f46111c33f